tasmota/tasmota_template.h

```diff
@@ -158,6 +158,7 @@ enum UserSelectablePins {
   GPIO_MAX7219CLK, GPIO_MAX7219DIN, GPIO_MAX7219CS, // MAX7219 interface
   GPIO_TFMINIPLUS_TX, GPIO_TFMINIPLUS_RX,  // TFmini Plus ToF sensor
   GPIO_ZEROCROSS,
+  GPIO_MCP2515_CS,                     // MCP2515 Chip Select
```

Comment:
For compatibility this needs to be moved to the end of the list

Reply:
Moved to above GPIO_INTERRUPT